### DISCUSSION TITLE
Does swc have official Chinese documentation?

### DISCUSSION THREAD

**Discussion by YanceyOfficial** [swc-project/swc#4966] [General]
If not, I can do it.

**Comment by kwonoj** [comment]
There is no official translations. We do not restrict userland translations, however given core team does not have a way to verify those translations we do not actively promote those as official docs

**Reply by YanceyOfficial** [reply]
Thx, I want to translate it for Chinese version. Do you think I should submit an i18n merge request to this repo, or create a new website like `zh-cn.swc.rs`?

**Reply by kwonoj** [reply]
Neither, as said we do not accept translations yet l. Also you can't use swc.rs subdomain since swc.rs is registered.

You can create your own domain if needed.
